ocs_ci/ocs/bucket_utils.py

```diff
@@ -2953,16 +2953,22 @@ def upload_obj_versions(mcg_obj, awscli_pod, bucket_name, obj_key, amount=1, siz
         mcg_obj (MCG): MCG object
         awscli_pod (Pod): Pod object where AWS CLI is installed
         bucket_name (str): Name of the bucket
-        obj_key (str): Object key
+        obj_key (str): S3 path to the object in the bucket
```

Comment:
Add `return` statement in doc string

Reply:
We're not actually returning anything since the merge of https://github.com/red-hat-storage/ocs-ci/pull/11322

Defining the empty etags list and returning it is a mistake in this PR - must have happened when I rebased this PR off of 11322. Thanks for bringing my attention to this - I'll fix it in a follow-up PR.

Reply:
Fixed in https://github.com/red-hat-storage/ocs-ci/pull/11438/commits/0eda7a886d95a7d462b578683e127fc71500d443